<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientClass</class>
 <widget class="QMainWindow" name="ClientClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>720</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/Client/Resources/scene.jpg);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>160</y>
        <width>361</width>
        <height>331</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/Client/Resources/face_1.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>160</y>
        <width>361</width>
        <height>331</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/Client/Resources/face_2.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>160</y>
        <width>361</width>
        <height>331</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/Client/Resources/face_3.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>160</y>
        <width>361</width>
        <height>331</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/Client/Resources/face_4.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_1">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>450</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Got it</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>450</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Got it</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>450</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Got it</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>450</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Got it</string>
      </property>
     </widget>
     <widget class="QPushButton" name="enterButton">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>580</y>
        <width>181</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ENTER</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="Face" name="face" native="true">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>640</y>
        <width>120</width>
        <height>80</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>150</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>200</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>250</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="Ball" name="ball" native="true">
      <property name="geometry">
       <rect>
        <x>1100</x>
        <y>330</y>
        <width>120</width>
        <height>80</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Face</class>
   <extends>QWidget</extends>
   <header>face.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Ball</class>
   <extends>QWidget</extends>
   <header>ball.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="client.qrc"/>
 </resources>
 <connections/>
</ui>
